content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -77,15 +73,15 @@ deployed Kubeflow will have administration privileges in the cluster.
 
 ### Pre-requisites: grant user minimal Kubernetes cluster access
 
-You must grant each user the minimal permission scope that allows them to 
+You must grant each user the minimal permission scope that allows them to
 connect to the Kubernetes cluster.
 
-For example, for Google Cloud Platform (GCP) users, you should grant the 
+For example, for Google Cloud users, you should grant the
```

Comment:
Here, it's `Google Cloud` with `Platform (GCP)` removed. I suggest that for consistency you could apply this in other files in this PR. I've highlighted a few of them for you to make it easier.